docs/quality-assuarance-guide.md

```diff
@@ -0,0 +1,144 @@
+# Quality Assurance Guide for Ballerina Extended Library Modules
+
+- Authors: @NipunaRanasinghe
+- Reviewers: @abeykoon @anupama-pathirage
+- Created: 2023/05/26
+- Updated: 2023/05/29
+
+## Introduction
+
+This document outlines a set of rules and guidelines to be followed during the development, testing, and documentation
+stages of the Ballerina extended library development lifecycle.
+By adhering to this checklist, Ballerina extended library developers can effectively evaluate and validate the
+functionality,
```

Comment:
Line break is intentional?

Reply:
Nope :) updated in https://github.com/ballerina-platform/ballerina-extended-library/pull/514/commits/f92f7644b6b5382042b36a9f8db861044363258d